From b6723caf5654c2c7cb4c39d66c679165b9105bab Mon Sep 17 00:00:00 2001
From: wep21 <daisuke.nishimatsu1021@gmail.com>
Date: Tue, 12 Sep 2023 16:50:56 +0900
Subject: [PATCH 2/2] disable lto

Signed-off-by: wep21 <daisuke.nishimatsu1021@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0dba58da..de3033b9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -49,7 +49,7 @@ endif()
 # --------------------------------------------------------
 if(UNIX)
   message(STATUS "GCC detected - Adding flags")
-  add_definitions(-Wall -Wextra)
+  add_definitions(-Wall -Wextra -fno-lto)
 
   set(ATOMIC_TEST_CODE "
       #include <atomic>
-- 
2.34.1

